src/components/attendee/RetunableMediaElement/types.tsx

```diff
@@ -0,0 +1,38 @@
+export enum RetunableMediaSource {
+  notTuned = "NOT_TUNED",
```

Comment:
Maybe something simpler like

```suggestion
  none= "NONE",
```

or `missing`, `empty` or anything without negative?

Would a `default` have some logic behind it?

Reply:
I think it's clearer that the default case is in line with what is displayed on the UI